apps/registration/src/hooks/__tests__/useUpdatePatient.test.tsx

```
@@ -141,229 +142,125 @@ describe('useUpdatePatient', () => {
-      const formDataWithFemaleGender = {
+    it('should upload photo when image is provided', async () => {
+      mockUpdateFhirPatient.mockResolvedValue(mockFhirPatientResponse);
+      mockUploadPatientPhoto.mockResolvedValue(undefined);
+
+      const formDataWithImage = {
         ...mockFormData,
-        profile: {
-          ...mockFormData.profile,
-          gender: 'female',
-        },
+        profile: { ...mockFormData.profile, image: 'data:image/jpeg;base64,abc123' },
```

ESLint

[error] Replace `·...mockFormData.profile,·image:·'data:image/jpeg;base64,abc123'` with `⏎··········...mockFormData.profile,⏎··········image:·'data:image/jpeg;base64,abc123',⏎·······`

(prettier/prettier)